src/controllers/user.js

```diff
@@ -0,0 +1,35 @@
+import { User } from '../database/models';
+import errorResponse from '../utils/error';
+import successResponse from '../utils/success';
+import imageUploader from '../helpers/photoUpload';
+exports.updateUserProfile=async (req,res)=>{
+const user=req.user.dataValues
+// console.log(req.user)
+        try {
+
+            if(req.files) {
+                const imageFile = await imageUploader(req);
+                req.body.image=imageFile.url
+            }
+            if(!req.body.image)
+            {
+            req.body.image="https://cdn.onlinewebfonts.com/svg/img_337050.png"
+            }
+            const {firstName,lastName,username,phoneNumber,image,gender,preferredLanguage,preferredCurrency,department} = req.body;
+            if(!firstName || !lastName || !username || !phoneNumber || !image || !gender || !preferredLanguage || !preferredCurrency || !department){
```

Comment:
can you use joi for validation, Please

Reply:
i dii sir check in validation folder and its work for really